package.json

```diff
@@ -37,17 +37,19 @@
     "lodash-cli": "4.12.0",
     "maki": "0.5.0",
     "minimist": "~1.2.0",
-    "mocha": "~2.3.4",
-    "mocha-phantomjs-core": "^1.3.0",
+    "mocha": "~2.5.3",
+    "mocha-phantomjs-core": "^2.0.1",
     "name-suggestion-index": "0.1.1",
     "request": "~2.16.2",
+    "rollup": "0.31.2",
     "sinon": "~1.6",
     "sinon-chai": "~2.3.1",
     "smash": "0.0",
     "svg-sprite": "1.2.19",
     "uglify-js": "~2.4.16",
     "xml2js": "~0.4.13",
-    "xmlbuilder": "~4.2.0"
+    "xmlbuilder": "~4.2.0",
+    "phantomjs-prebuilt": "2.1.7"
```

Comment:
Latest commit removes the travis manual build pattern - we'll see if that works.